src/NuGet.Core/NuGet.Packaging/Signing/Cms/ManagedCmsWrapper.cs

```diff
@@ -47,7 +47,10 @@ public void AddCertificates(IEnumerable<X509Certificate2> certificates)
         {
             foreach (var cert in certificates)
             {
-                _signedCms.AddCertificate(new X509Certificate2(cert));
+                using (var certificate = new X509Certificate2(cert))
+                {
+                    _signedCms.AddCertificate(certificate);
+                }
```

Comment:
By putting this is a using block, it means the certificate that was added to `_signedCms` is now disposed. Are you sure that's ok?

Reply:
Looks like the `AddCertificate `method will create a copy of the certificate in following code:
https://github.com/dotnet/runtime/blob/master/src/libraries/System.Security.Cryptography.Pkcs/src/System/Security/Cryptography/Pkcs/SignedCms.cs#L639-L671

Reply:
Consider adding a comment saying that `AddCertificates` copies the cert.

Reply:
So it should be ok not new any certificate, just use the one from parameter. 
I'll fix it. Thanks!